cypress/utils/utils.ts

```diff
@@ -835,6 +838,35 @@ export function createMultipleApplications(numberofapplications: number): Array<
     return applicationList;
 }
 
```

Comment:
Added createMultipleApplication method with Business Service and Tags, Please Review !